.env

```diff
@@ -40,7 +40,7 @@ PROVIDER_MULTIPART_EDGE=mixed
 PROVIDER_DATA_APP_ENDPOINT=https://be-dataapp-provider:8083/data
 # In case of WSS configuration
 #PROVIDER_DATA_APP_ENDPOINT=https://be-dataapp-provider:9000/incoming-data-app/routerBodyBinary
-PROVIDER_IDSCP2=false
+PROVIDER_IDSCP2=true
```

Comment:
why do we have default value set to true for idscp?

Reply:
Probably wrong config from my side, and committed like that.
I will change it to false